plugins/esi/combo_handler.cc

```diff
@@ -1052,21 +1095,86 @@ getContentType(TSMBuffer bufp, TSMLoc hdr_loc, string &resp_header_fields)
     int n_values = TSMimeHdrFieldValuesCount(bufp, hdr_loc, field_loc);
     for (int i = 0; i < n_values; ++i) {
       value = TSMimeHdrFieldValueStringGet(bufp, hdr_loc, field_loc, i, &value_len);
-      if (!values_added) {
-        resp_header_fields.append("Content-Type: ");
-        values_added = true;
-      } else {
-        resp_header_fields.append(", ");
+      ts::TextView tv{value, value_len};
+      tv              = tv.prefix(';').rtrim(std::string_view(" \t"));
+      auto const &ctw = _content_type_whitelist;
```

Comment:
What is the point of `ctw`? Why not use `_content_type_whitelist`?

Reply:
You got it SWOV.